docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -2157,53 +2157,127 @@ limitation.
 2.3.15 Employment (20)
 ~~~~~~~~~~~~~~~~~~~~~~
 
-To represent businesses and employment dynamics we will use another
-directed tripartite graph (analogous to our migration component),
-showing arcs from simulants (part A) to employers (part B) as well as
-arcs from employers to their addresses (part C).
-
-This construct allows us to represent businesses that employ one or
-more people, as well as individuals who are employed by multiple
-businesses.  We will also be able to add business dynamics in the
-future, e.g. new businesses arriving, old businesses closing down, and
-even merges, as well as name changes and address changes.  All of this
-will go into our simulated tax return data, which we must make a
-scheme for before we access restricted tax data (since even the schema
-of this data is restricted information).
-
-To keep things simple for starters, we will give everyone age 18 and over a
-random edge to an employer, chosen from a skewed distribution to
-ensure that there are a few large employers and a "fat tail" of small
-employers. We will change employers randomly at the rate of 50 changes
-per 100 person years, and change employer addresses at a rate of 10
-changes per 100 person years.  For now, we will have distinct
-addresses for businesses and households, but eventually we might want
-to intentionally include duplicates, e.g. if someone operates a
-business out of their home.
-
-To keep things simple, for now when businesses move to a new address,
-it will be a totally new address. No household or business will ever
+.. note::
+
+  Employment is one of the less-developed parts of our model.
+  We have included the aspects that seem particularly relevant to person linkage.
+  With this employment model, business linkage in the resulting data
+  is likely missing key real-life challenges.
+
+We consider all simulants age 18 and over to be working-age; all such
+simulants either have an employer or are considered unemployed.
+We only allow a single employer at a time for each simulant.
+
+Each employer has a single street address.
+
+Initialization
+^^^^^^^^^^^^^^
+
+We initialize 58% of the working-age population to be unemployed,
```

Comment:
57.6%. we technically assign "unemployed" with probability .576, so it won't be precisely this number.